src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -64,7 +64,7 @@ const Body = BaseForm.extend({
     return uiSchemas.concat([
       {
         name: 'confirmPassword',
-        label: loc('oie.password.confirmPasswordLabel','login'),
+        label: loc('password.confirmPassword.placeholder','login'),
```

Comment:
String within brackets is customized 

Before : 

![Screen Shot 2021-05-26 at 11 02 12 AM](https://user-images.githubusercontent.com/23267876/119745790-b2fddd00-be43-11eb-8d73-c1e3d00c0229.png)

After: 
![Screen Shot 2021-05-26 at 11 01 48 AM](https://user-images.githubusercontent.com/23267876/119745795-b6916400-be43-11eb-914a-8ae00e69c018.png)

Reply:
This seems to change the default text from `Re-enter Password` to `Repeat password` for OIE orgs.
* We will need @vincentvoong-okta 's approval on this one.
* Btw, FYI @vincentvoong-okta  - This will provide V1 parity in terms of text customization for the re-enter field.

Reply:
Sure, FYI this requirement was discussed with to support customization @unniravindranathan-okta

Reply:
Time permitting, we should move all altered text to another properties file, which takes precedent over `login.properties`.

This way, OIE string values can override v1 resources without changing the key name or dropping customizations.

Reply:
Going to revert this particular change, after talking to PM

Reply:
Created a ticket to track the new bundle https://oktainc.atlassian.net/browse/OKTA-399751